src/com/flower/kiev/FlowerStore.java

```diff
@@ -0,0 +1,79 @@
+package com.flower.kiev;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class FlowerStore {
+
+    private int wallet;
+    private Flower[] flowers;
+
+    public int getWallet() {
+        return wallet;
+    }
+
+    public void printFlowers() {
+        for (int i = 0; i < flowers.length; i++) {
+            System.out.print(flowers[i].getClass().getSimpleName() + " ");
+        }
+    }
+
+    public Flower[] sell(int numRose, int numChamomile, int numTulip) {
+        int numOfFlowers = numRose + numChamomile + numTulip;
+        flowers = new Flower[numOfFlowers];
+
+        for (int i = 0; i < numRose; i++) {
```

Comment:
![image](https://user-images.githubusercontent.com/20267148/54381882-7ba61a00-4697-11e9-985e-b5f7f05271ce.png)
You create two objects but really use only one.
It need to be changed. And it would be samething like:
```
 flowers[i] = new Rose();
 wallet += flowers[i].gitPrice();
```